### PR TITLE
What's new updates /8

### LINKED ISSUE #30982: Feedback : AsParameters  and move the “Improved performance using Interceptors”
### Description

-	We can problem remove the “Support for AsParameters and automatic metadata generation” heading under minimal APIs since that was not a net-new feature in .NET 8. It was part of the overall work done to add compile-time support.
-	I would move the “Improved performance using Interceptors” section to be under “Request delegate generator” since interceptors are an aspect of the request delegate generator and not an independent feature in and off themselves.

cc @tdykstra 

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-8.0?view=aspnetcore-7.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/release-notes/aspnetcore-8.0.md

### Document ID

f30dbb9e-d64d-a925-01ca-00d9d035e6c1

### Article author

Rick-Anderson

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30982

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/release-notes/aspnetcore-8.0.md](https://github.com/dotnet/AspNetCore.Docs/blob/35dc991c3bf13bf87b3e4e0660eeb306882794b6/aspnetcore/release-notes/aspnetcore-8.0.md) | [What's new in ASP.NET Core 8.0](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-8.0?branch=pr-en-us-30983) |

### CODE REVIEW

File: aspnetcore/release-notes/aspnetcore-8.0.md [hunk 1]
```diff
@@ -496,12 +496,6 @@ The antiforgery token is only validated if:
 
 For more information, see [Antiforgery with Minimal APIs](xref:security/anti-request-forgery?view=aspnetcore-8.0&preserve-view=true#afwma).
 
-### Support for `AsParameters` and automatic metadata generation
-
-Minimal APIs generated at compile-time include support for parameters decorated with the [`[AsParameters]`](xref:Microsoft.AspNetCore.Http.AsParametersAttribute) attribute and support automatic metadata inference for request and response types. Consider the following code:
-
-:::code language="csharp" source="~/release-notes/sample/ProgramAsParameters.cs" highlight="22":::
-
 The preceding generated code:
```
Comment (tdykstra) [inline aspnetcore/release-notes/aspnetcore-8.0.md:499]: Now it looks like you've got content talking about a snippet that is no longer there.

File: aspnetcore/release-notes/aspnetcore-8.0.md [hunk 2]
```diff
@@ -584,6 +574,10 @@ We're working to ensure that as many as possible of the Minimal API features are
 
 The RDG is enabled automatically in a project when publishing with native AOT is enabled. RDG can be manually enabled even when not using native AOT by setting `<EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator>` in the project file. This can be useful when initially evaluating a project's readiness for native AOT, or to reduce the startup time of an app.
 
+### Improved performance using Interceptors
+
+The Request Delegate Generator uses the new [C# 12 interceptors compiler feature](/dotnet/csharp/whats-new/csharp-12) to support intercepting calls to minimal API’s [Map](xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions) action methods with statically generated variants at runtime. The use of interceptors results in increased startup performance for apps compiled with `PublishAot`.
```
Comment (tdykstra) [inline aspnetcore/release-notes/aspnetcore-8.0.md:579]: Remove curly quote

Are the minimal API Map methods really called action methods?

```suggestion
The Request Delegate Generator uses the new [C# 12 interceptors compiler feature](/dotnet/csharp/whats-new/csharp-12) to support intercepting calls to minimal API [Map](xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions) action methods with statically generated variants at runtime. The use of interceptors results in increased startup performance for apps compiled with `PublishAot`.
```

**Review by tdykstra — APPROVED** [review]
Looks good, but a couple of issues to think about.

**Comment by Rick-Anderson** [comment]
@tdykstra  removed action and "the preceding code ..."

**Pull request merged**